src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/Common.ps1

```diff
@@ -223,43 +223,19 @@ Gets the values of the parameters used in the Server Key Vault Key tests
 #>
 function Get-SqlServerKeyVaultKeyTestEnvironmentParameters ()
 {
-	return @{ rgName = Get-ResourceGroupName;
-			  serverName = Get-ServerName;
-			  databaseName = Get-DatabaseName;
-			  keyId = "https://akvtdekeyvault.vault.azure.net/keys/key1/51c2fab9ff3c4a17aab4cd51b932b106";
-			  serverKeyName = "akvtdekeyvault_key1_51c2fab9ff3c4a17aab4cd51b932b106";
-			  vaultName = "akvtdekeyvault";
-			  keyName = "key1"
-			  location = "centraluseuap";
+	return @{ rgName = "ps-test";
+			  serverName = "ps-tdeakv-svr";
+			  databaseName = "ps-tdeakv-db";
+			  vaultName = "ps-vault";
+			  keyName = "ps-key";
+			  location = "westeurope";
+			  keyId = "https://ps-vault.vault.azure.net/keys/ps-key/65011ae18b5747eba29ea8e5fc85c1a9";
+			  serverKeyName = "ps-vault_ps-key_65011ae18b5747eba29ea8e5fc85c1a9";
```

Comment:
I removed any changes relating to updating the ServerKey and EncryptionProtector related cmdlets to the Swagger generated SDK, so that this change is only related to Identity changes for Server.